SETUP/tests/ZipValidateTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
```

Comment:
Is there a reason we didn't add these functions to `misc_ZipMethodsTest.php`? If we want them in a different file it should follow the `misc_<class>.php` naming convention.

Reply:
The reason is that when I tried to run the `misc_ZipMethodsTest.php` alone from the command line phpUnit said 
> Class misc_ZipMethodsTest could not be found in /home/rp31/public_html/u-rp75-dproofreaders/SETUP/tests/misc_ZipMethodsTest.php

The documentation says
> When invoked as shown above, the PHPUnit command-line test runner will look for a ArrayTest.php source code file in the current working directory, load it, and expect to find an ArrayTest test case class. It will then run the tests found in that class.

so I thought the easiest way round the problem was to make a new file. But, as you say, it would be better to put them in the existing file. Is there away to run it from the command line?

Reply:
Huh. I think the older version of phpunit we started with worked differently (you could just give it the filename and it would just run the tests in that file). You can run specific classes like this:
```
../../vendor/bin/phpunit --filter MiscUtils
```